test/src/test/java/hudson/cli/AbstractBuildRangeCommand2Test.java

```diff
@@ -88,7 +90,7 @@
 
     @Test public void dummyRangeShouldSuccessEvenTheBuildIsRunning() throws Exception {
         FreeStyleProject project = j.createFreeStyleProject("aProject");
-        project.getBuildersList().add(new Shell("echo 1\nsleep 10s"));
+        project.getBuildersList().add(Functions.isWindows() ? new BatchFile("echo 1\r\nping -n 10 127.0.01 >nul") : new Shell("echo 1\nsleep 10s"));
```

Comment:
That doesn't look like an IP address to me.  Didn't you mean `127.0.0.1` instead of `127.0.01`?